katana/config/newtestwrapperfile.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" ?>
```

Comment:
Is the required file ?  File is already present under Warriorspace.

Reply:
Hi @santhosh-warrior ,

we are using this file as a reference(tag names, setup and cleanup blocks) while generating wrapper file. We cannot use wrapper file available in the warriorspace/wrapperfiles as the content available in the tw_sample will be shown as default in UI.